Clinical-Genomics/cg
# adds cli/set/microbial_order

This PR adds possibility to change apptag on microbial orders.

How to test:
0. install on stage rasta
1. us
1. cg set microbial_order -a [apptag] [order_id] [user_signature]

Expected outcome:
* All samples that were changed should have been listed
* the apptag changed, 
* a comment added to the sample explaining what changed together with the user signature
